src/app/components/sidebar/filters/Filters.tsx

```diff
@@ -1,29 +1,65 @@
 import * as Styled from './Filters.styles';
-import { useIntl } from 'react-intl';
 import { Condition } from './condition/Condition';
 import { LabelWithButton } from 'ui';
 import { FiltersProps } from './Filters.types';
+import { Price } from './price/Price';
+import { useLocale } from 'hooks';
+import { FormatNumberOptions } from 'react-intl';
+
+export const Filters = ({ onConditionClick, onPriceChange, activeFilters }: FiltersProps) => {
+  const { formatMessage, formatNumber } = useLocale();
+
+  const currencyConfig: FormatNumberOptions = {
+    style: 'currency',
+    currency: 'PLN',
+    maximumFractionDigits: 0,
+  };
 
-export const Filters = ({ onConditionClick, activeFilters }: FiltersProps) => {
-  const { formatMessage } = useIntl();
   return (
     <>
-      {activeFilters.condition && activeFilters.condition.length > 0 && (
+      {((activeFilters.condition && activeFilters.condition.length > 0) ||
+        activeFilters.minPrice ||
+        activeFilters.maxPrice) && (
```

Comment:
Ło kurka... a jesteś w stanie wyciągnąć te wszystkie conditiony jako jedna zmienna przed returna i jakoś to nazwać? Na tym polega pisanie samokomentującego się kodu, tzn że jeśli jest jakiś dziwny twór, to się go wyciąga jako zmienną/funkcję/hooka i odpowiednio nazywa